intent_classification/cnn/train_with_pretrained_embeddings.py

```diff
@@ -0,0 +1,100 @@
+import tensorflow as tf
+import math
+import sys
+sys.path.append("../")
+import atis_data
+
+FLAGS = tf.app.flags.FLAGS
```

Comment:
I wasn't sure on the use of `FLAGS` and then I came across [this](https://stackoverflow.com/a/33938519). 

Basically, `FLAGS` are just supposed to be used for demo apps. It's undocumented and the APIs might change in the future. We should be fine for this project, but as a general practice I think we should avoid this. 

Should we change or leave it as is? Do you have any other ways of doing it apart from just declaring a global object?

Reply:
hmm. Lets live with FLAGS